Khalil/khalil_PrimeNumber.java

```diff
@@ -0,0 +1,24 @@
+package Khalil;
+
+public class khalil_PrimeNumber {
+
+    public static void main(String[] args) {
+
```

Comment:
There is a bug in your code!!!! your didn't cover negative numbers and number 1 itself.